### PR TITLE
Feature/topic configuration

### PR DESCRIPTION
**Pull request by jacobcrawford**
**Description of your changes:**
We enable a KafkaTopic to be configurable according to standard Kafka topic configurations.

Checklist:

* [x] I have bumped the chart version
* [x] I have documented my changes if this is needed
* [x] I have described my changes in the "description of your changes" field
* [ ] I have tested this change on a running cluster and the Argocd dashboard shows healthy and synced
* [ ] I have created the necessary tests in the chart, if they are possible/necessary
* [x] I have squashed commits if necessary

### CODE REVIEW

**Comment by jacobcrawford** [comment]
Configs get propagated to KafkaTopic:
image

**Comment by klausenbusk** [comment]
Hmm, is having the version bump in a separate commit worth it? It basically just adds noise?

**Review by skovbakke — DISMISSED** [review]
LGTM 🥳

File: charts/kafka/chart/templates/kafka-topic-template.yaml [hunk 1]
```diff
@@ -9,5 +9,8 @@ spec:
   topicName: {{ .name }}
   partitions: {{ .partitions }}
   replicas: {{  .replicas }}
+  {{- if .config }}
+  config: {{ .config | toYaml | nindent 4 }}
+  {{ end }}
```
Comment (jaxels10) [inline charts/kafka/chart/templates/kafka-topic-template.yaml:14]:
```suggestion
  {{- end }}
```
I know this is a small thing, but the templated files will look much better if we remove unnecessary white space

File: charts/kafka/chart/templates/kafka-topic-template.yaml [hunk 2]
```diff
@@ -9,5 +9,8 @@ spec:
...
 ---
 {{ end }}
```
Comment (jaxels10) [inline charts/kafka/chart/templates/kafka-topic-template.yaml:16]:
```suggestion
{{- end }}
```

**Review by jaxels10 — CHANGES_REQUESTED** [review]
Very very small changes. It generally looks good to me, although we should probably try to reduce whitespace in our templated files.

**Review by jaxels10 — APPROVED** [review]
LGTM

**Pull request merged**